Testing/SystemTests/tests/analysis/ISIS_WISHDiffractionFocussing.py

```diff
@@ -160,7 +160,7 @@ def validate(self):
         self.assertEqual(self.table.rowCount(), 8)
         self.assertEqual(len(self.output_names), 8)
         self.tolerance = 1e-4
```

Comment:
Do you want to try moving this value back to `1e-5` as it was before #24829?

Reply:
I'll check if that works now

Reply:
And now things start to make sense...

Reply:
it appears to work fine